.github/workflows/api-release.yml

```
@@ -0,0 +1,38 @@
+runs-on: ubuntu-latest
+steps:
+  - name: Check out Git repository
+    uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)